assignments/function-conversion.js

```diff
@@ -5,26 +5,53 @@
 // };
 // myFunction();
 
+const myFunction = () => {
+	console.log("Function was invoked!");
+};
+
+console.log(myFunction());
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+const anotherFunction = (param) => {
+	return param + ' it worked!';
+};
+
+console.log(anotherFunction('Word,'));
+
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+const add = (param1, param2) =>  param1 + param2;
+
+console.log(add(5318007, 1));
```

Comment:
Yeeees, very nice with the implicit return here. 👍

Reply:
I love that! It is super clean looking. I also figured out the stretch challenge in one attempt some how! Wouldn't have thought so yesterday.